servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -56,10 +56,10 @@
           LINKED_HASH_MAP_INIT_LOAD_FACTOR, LINKED_HASH_MAP_ACCESS_ORDERED));
 
   public LRUEvictor(BlockMetadataManager meta) {
-    mMeta = Preconditions.checkNotNull(meta);
+    mMetaManager = Preconditions.checkNotNull(meta);
```

Comment:
We can probably move this check into the factory method in another PR, I will open a ticket.